tips/TIP-0022/tip-0022.md

```diff
@@ -29,7 +29,7 @@ Various other protocol TIPs rely on certain constants that need to be defined fo
 | Min PoW Score                         | 1500.0           | Minimum PoW score for blocks to pass syntactic validation.                                                                                                       |
 | First Milestone Index                 | 1                | First valid milestone index.                                                                                                                                     |
 |                                       |                  |                                                                                                                                                                  |
-| Max IOTA Supply                       | 4600000000000000 | Total amount of IOTA coins in circulation.                                                                                                                       |
+| Max IOTA Supply                       | 4600000000,000000 | Total amount of IOTA coins in circulation.                                                                                                                       |
```

Comment:
Token Supply is an integer, while this suggests it is a floating point number. #139 changes the description to micros which is better imo.